lib/Mailer/Methods/ErrorMappers/BlacklistErrorMapperTrait.php

```diff
@@ -0,0 +1,14 @@
+<?php
+namespace MailPoet\Mailer\Methods\ErrorMappers;
+
+use MailPoet\Mailer\MailerError;
+use MailPoet\Mailer\SubscriberError;
+use MailPoet\WP\Functions as WPFunctions;
+
+trait BlacklistErrorMapperTrait {
+  function getBlacklistError($subscriber) {
+    $message = sprintf(WPFunctions::get()->__('%s has returned an unknown error.', 'mailpoet'), self::METHOD);
```

Comment:
Using `self::METHOD` in a trait without defining it is really not nice 😞

Reply:
Traits cannot have constants in PHP. We don't support custom sending methods so this is likely under control.